arrow-libs/core/arrow-meta/build.gradle

```diff
@@ -30,8 +30,4 @@ dependencies {
     testImplementation "io.kotlintest:kotlintest-runner-junit4:$KOTLIN_TEST_VERSION", excludeArrow
     def toolsJar = Jvm.current().getToolsJar()
 
-    if (!toolsJar)
-        throw new GradleException("tools.jar not found at your JAVA_HOME dir ${Jvm.current().getJavaHome().getAbsolutePath()}.\n" +
-                "Building with a JRE or JDK9 is currently not supported.")
-    testCompileOnly files(toolsJar)
```

Comment:
Just a note to remember recovering this local "fix" for your local env.

Reply:
Perfect, thanks @rachelcarmena  ;)